Added Norwegian Bokmål translations

master3395 opened the pull request:
Add Norwegian Bokmål

## What's changed?
*Briefly describe the changes you've made here*

Added Norwegian Bokmål.

adhirajsinghchauhan reviewed (changes requested):
1. Strings from line 185 onwards haven't been translated yet. Do you intend to complete them?
2. Remove all strings marked with `translatable="false"`
3. Remove all `TODO` comments if the corresponding strings have been translated
4. Fix build error: some strings have unescaped apostrophes (e.g. line 15: there's a space between `\` and `'` - remove the space)

master3395 commented:
1. All is translated.
2. Removed all strings with translatable="false"
3. Removed TODO comment
4. Fixed strings that were found.

Translated by:

🐧Master3395🐧#0183 (Discord) 80357836940382208
Taco#1219 (Discord) 189807821364264961
